### PR TITLE
chore: add SUPPORT.md for escalation guidelines

### PR DESCRIPTION
**Pull request by schristoff**

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Support Guidelines
+
+We strive to create clear guidelines on communication to the Zarf team to provide a good community experience.
+
+
+## Standard Process
+All questions, and code issues should be a [Github Issue](https://github.com/defenseunicorns/zarf/issues/new/choose) that follows the issue template. 
+
+Following the templates provides the Zarf community a foundation of understanding to be able to assist quickly.
+After an issue is made, this issue can be brought into other chanels such as the [Kubernetes Slack #Zarf](https://zarf.dev/slack) channel or the [bi-weekly Zarf Community Meeting](zarf.dev/???).
```
Comment (schristoff) [inline SUPPORT.md:10]: @salaxander  - When the biweekly community meetings get set up, I can update this on the zarf.dev/support page?

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                Github Issue 
+                /          \
+    Zarf Slack Channel    Zarf Community Call
+
+
+
+## Sensitive Information Process
+For issues from those who are unable to post on Github, you may send an email using the following issue template filled out to zarf-private@googlegroups.com](zarf-dev-private@googlegroups.com).
+
+The response time to emails may be delayed  as they are not able to receive community help, so we encourage participation into Github Issues as much as possible.
```
Comment (AustinAbro321) [inline SUPPORT.md:21]:
```suggestion
For issues from those who are unable to post on Github, you may send an email using the following issue template filled out to [zarf-private@googlegroups.com](zarf-dev-private@googlegroups.com).

The response time to emails may be delayed as they are not able to receive community help, so we encourage participation into Github Issues as much as possible.
```

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Support Guidelines
+
+We strive to create clear guidelines on communication to the Zarf team to provide a good community experience.
+
+
+## Standard Process
+All questions, and code issues should be a [Github Issue](https://github.com/defenseunicorns/zarf/issues/new/choose) that follows the issue template.
```
Comment (AustinAbro321) [inline SUPPORT.md:7]: Thoughts on adding a hook here for checking out the docs first? Not sure exactly the best way to word it

**Review by lucasrod16 — APPROVED** [review]

**Comment by schristoff** [comment]
whoops #2480

**Pull request closed without merging**